docs/en/observability/rate-aggregations.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[rate-aggregations]]
+= Rate aggregations
+
+You can use a rate aggregation to analyze the rate at which a specific field changes over time.
+This type of aggregation is useful when you want to analyze fields like counters.
+
+For example, imagine you have a counter field called restarts that increments each time a service restarts.
+You can use rate aggregation to get an alert if the service restarts more than X times within a specific time window (for example, per day).
+
+[discrete]
+[[how-rates-are-calculated]]
+== How rates are calculated
+
+Rates used in alerting rules are calculated by comparing the maximum value of the field in the previous bucket to the maximum value of the field in the current bucket and then dividing the result by the number of seconds in the selected interval.
+For example, if the value of the restarts increases, the rate would be calculated as:
+
+`(max_value_in_current_bucket - max_value_in_previous_bucket)/interval_in_seconds`
+
+In this example, let’s assume you have one document per bucket with the following data:
+
+[source,json]
+----
+{
+"timestamp": 0000,
+"restarts": 0
+}
+
+{
+"timestamp": 60000,
+"restarts": 1
+}
+
+----
+
+// I might be a little timestamp naive, but I don't understand why we start with 0000 in this example (or whether it matters either way).
+
+Let’s assume the timestamp is a UNIX timestamp in milliseconds,
+and we started counting on Thursday, January 1, 1970 12:00:00 AM.
+In that case, the rate will be calculated as follows:
+
+`(max_value_in_current_bucket - max_value_in_previous_bucket)/interval_in_seconds`, where:
+
+* `max_value_in_current_bucket` [now-1m → now]: 1
+* `max_value_in_previous_bucket` [now-2m → now-1m]: 0
+* `interval_in_seconds`: 60
+
+The rate calculation would be: `(1 - 0) / 60 = 0.0166666666667`
+
+If you want to alert when the rate of restarts is above 1 within a 1-minute window, you would set the threshold above `0.0166666666667`.
+
+The calculation you need to use depends on the interval that's selected:
```

Comment:
Personally, I don't think the information after this line is that useful (I've added them initially more for brainstorming)

@jasonrhodes @dedemorton do you see keeping the rest of the section (from this line onward) useful or shall we remove it? (Especially, the Network rate example would be useful if we add more context, having only an image seems more confusing to me.)

Reply:
I'd opt for removing things we aren't sure of and evolving the docs more as we go. I think what's above this section, and the rest of the changes in this PR, are already a huge improvement, so I don't think we lose much if we pull back a bit on some of the things we haven't had time to fully edit/tweak/etc.

Reply:
>I don't think the information after this line is that useful

I agree. TBH, I had to read the info in the table a couple times to understand why it was there, and it seems to restate what we've already said, only in a slightly different way.

Regarding the network rate example, I agree that having a screen capture is not quite good enough, but I do think it's useful to show a realistic example. It's especially important to provide more context for accessibility. That's why I've added a long alt text description to the image, but it would be much better to properly introduce the example and provide more context. 

@maryam-saeidi What additional context should we provide here? Should we show an example document? Describe what's shown in the screen capture? What else? I'm happy to fill in missing details if you can provide them, or I can remove the screenshot for now, and we can revisit this later. WDYT?